conductr_cli/sandbox_run_jvm.py

```diff
@@ -235,6 +235,22 @@ def validate_jvm_support():
         raise JavaCallError('Failure calling `java -version`')
 
 
+def validate_hostname_lookup():
+    """
+    Validates if the hostname lookup by Java is slow given the issue https://github.com/akka/akka/issues/22160
+    Validation fails if the hostname is not part of the /etc/hosts file and the OS is macOS.
+    A HostnameLookupError is raised if the validation fails
+    """
+    if host.is_macos():
```

Comment:
We validate the Java version prior to this check. So if the Java version is not valid then we do not we will abort before we perform the hostname lookup validation.